spec.html

```diff
@@ -471,8 +471,24 @@ <h1>The Syntactic Grammar</h1>
 
     <emu-clause id="sec-grammar-notation" namespace=grammar-notation>
       <h1>Grammar Notation</h1>
-      <p>Terminal symbols of the lexical, RegExp, and numeric string grammars are shown in `fixed width` font, both in the productions of the grammars and throughout this specification whenever the text directly refers to such a terminal symbol. These are to appear in a script exactly as written. All terminal symbol code points specified in this way are to be understood as the appropriate Unicode code points from the Basic Latin range, as opposed to any similar-looking code points from other Unicode ranges.</p>
-      <p>Nonterminal symbols are shown in <i>italic</i> type. The definition of a nonterminal (also called a &ldquo;production&rdquo;) is introduced by the name of the nonterminal being defined followed by one or more colons. (The number of colons indicates to which grammar the production belongs.) One or more alternative right-hand sides for the nonterminal then follow on succeeding lines. For example, the syntactic definition:</p>
+      <p>The nonterminal symbols of all the ECMAScript grammars are shown in <i>italic</i> type.</p>
+      <p>In all the grammars, 'verbatim' terminal symbols are shown in `fixed-width` font. These are to appear in a source text exactly as written. All terminal symbol code points specified in this way are to be understood as the appropriate Unicode code points from the Basic Latin range, as opposed to any similar-looking code points from other Unicode ranges.</p>
+      <p>In grammars whose terminal symbols are individual Unicode code points (i.e., the lexical, RegExp, and numeric string grammars), a contiguous run of multiple fixed-width code points appearing in a production is a simple shorthand for the same sequence of code points, written as standalone terminal symbols.</p>
+      <p>For example, the production:</p>
+      <emu-grammar type="example">
+        HexIntegerLiteral :: `0x` HexDigits
+      </emu-grammar>
+      <p>is a shorthand for:</p>
+      <emu-grammar type="example">
+        HexIntegerLiteral :: `0` `x` HexDigits
+      </emu-grammar>
+      <p>Conversely, in the syntactic grammar, a contiguous run of fixed-width code points is a single terminal symbol.</p>
+      <p>Terminal symbols come in two other forms:</p>
+      <ul>
+        <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "&lt;FOO&gt;" where "FOO" is a mnemonic for the code point. These forms are defined in <emu-xref href="#sec-unicode-format-control-characters" title></emu-xref> and <emu-xref href="#sec-white-space" title></emu-xref>.</li>
+        <li>In the syntactic grammar, certain terminal symbols (e.g. |IdentifierName| and |RegularExpressionLiteral|) are shown in italics, as they refer to the nonterminals of the same name in the lexical grammar.</li>
```

Comment:
> (e.g. |IdentifierName| and |RegularExpressionLiteral|)

Are these going to create undue references from the Grammar Notation section? If so, is there a way to avoid that?

Reply:
> Are these going to create undue references from the Grammar Notation section?

Probably. Note that there's already 2 "references to #prod-IdentifierName" from Grammar Notation (one from inside a production, one from inside a paragraph), so I don't imagine these references would be exempt.

>  If so, is there a way to avoid that?

That sounds like a question for @bterlson. (I imagine one could mark them up so that they get styled appropriately but aren't picked up by ecmarkup's reference-detector.)

Reply:
By "undue references", you mean IdentifierName and RegularExpressionLiteral will link to the example definitions in this clause rather than the non-namespaced definition? If so, there is no way to opt-out of this behavior presently :( Happy to take a proposal for how to do it tho!

Reply:
Unlikely that was meant, because this PR doesn't add example definitions for IdentifierName and RegularExpressionLiteral.

But since you bring it up, it seemed to me that ecmarkup was smart enough not to do that. E.g., consider this clause's example definitions for _ArgumentList_ and _VariableDeclaration_. If you go to occurrences of those nonterminals *outside* this clause, the link will take you to the proper definition, *not* the example definition in this clause.

Reply:
> Are these going to create undue references from the Grammar Notation section?

In the deploy preview those references do indeed exist, but I don't really see them as undue. I'd be more surprised to have the link exist only in one direction (so that these link to their definitions, but the definitions do not link to these references).